versions-manifest.json

```diff
@@ -0,0 +1,181 @@
+[
```

Comment:
I think this file should be renamed and placed to some subfolder rather than root of repository. Or placed into separate repo `actions/java-versions`